src/load-glb-and-screenshot.js

```diff
@@ -1,8 +1,9 @@
-module.exports = async (page, {glbPath, outputPath, format, quality, timeout}) => {
-  return new Promise(async (resolve, reject) => {
-    page.exposeFunction('resolvePromise', resolve);
-    page.exposeFunction('rejectPromise', reject);
-    await page.evaluate(async (browser_glbPath, browser_outputPath, browser_format, browser_quality, timeout) => {
+module.exports = (page, {glbPath, outputPath, format, quality, timeout}) => {
+  page.exposeFunction('resolvePromise', Promise.reject);
+  page.exposeFunction('rejectPromise', Promise.resolve);
+
+  return page.evaluate((browser_glbPath, browser_outputPath, browser_format, browser_quality, timeout) => {
```

Comment:
`page.evaluate` returns a Promise which resolves to one of the following:

1. If the function passed to the `page.evaluate` returns a Promise, then page.evaluate would wait for the promise to resolve and return its value.
2. If the function returns a serializable value, it will return that
3. If the function returns a non-serializable value, it will return undefined.

https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pageevaluatepagefunction-args